gnomad/resources/grch37/gnomad.py

```diff
@@ -104,6 +104,29 @@ def _liftover_data_path(data_type: str, version: str) -> str:
     return f"gs://gnomad-public-requester-pays/release/{version}/liftover_grch38/ht/{data_type}/gnomad.{data_type}.r{version}.sites.liftover_grch38.ht"
 
 
+def _public_constraint_ht_path(version: str) -> str:
+    """
+    Get public constraint table path.
+
+    :param version: One of the release versions of gnomAD on GRCh38
+    :return: Path to constraint Table
+    """
+    return f"gs://gnomad-public-requester-pays/release/{version}/constraint/gnomad.v{version}.lof_metrics.by_transcript.ht"
+
+
+def _public_pext_path(data_format: str) -> str:
```

Comment:
Let's not worry about the tsv here, just want to be able to load the ht

```suggestion
def _public_pext_path() -> str:
```